src/components/CreateResolution/CompleteResolution.vue

```diff
@@ -364,7 +363,10 @@ export default class CompleteResolution extends Mixins(CommonMixin, DateMixin, E
   readonly ItemTypes = ItemTypes
   readonly CorpTypeCd = CorpTypeCd
   readonly BulletListTypes = BulletListTypes
+
+  // Validation Rules
   readonly Rules = Rules
+  readonly RuleHelpers = RuleHelpers
```

Comment:
You only need to "re-export" this if it's used in the template. No change require here -- it's better to be consistent with re-exports (rules, enums, etc).

Reply:
ah ic.  will remove that then.

Reply:
Up to you. It's not needed, but on the other hand consistency (ie, re-export everything) is good. 🤷‍♂️

Reply:
Removed re-export.  Figured we shouldn't keep things around that aren't used/needed.